language_generators/scala-defref-extractor/src/main/scala/io/bazeltools/buildgen/shared/PathTree.scala

```diff
@@ -76,60 +92,65 @@ object PathTree {
         case head :: tail =>
           branches.get(head) match {
             case Some(n) => n.getAt(tail)
-            case None    => None
+            case None => None
```

Comment:
I didn't run it because I was afraid it would rewrite everything, but I should have just tried after a commit. I see that you have run it already so it doesn't rewrite the repo.

We should probably add a CI test to require it.